backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -153,7 +155,48 @@ private Specification<TestEvent> buildTestEventSearchFilter(
       return p;
     };
   }
+  // methods to replace the methods that return List<TestEvent> once we get
+  // backwards compability issues resolved
+  @Transactional(readOnly = true)
+  @AuthorizationConfiguration.RequirePermissionReadResultListAtFacility
+  public Page<TestEvent> getTestEventsResultsPage(
```

Comment:
Similar to @emmastephenson's point for the naming of `getAllFacilityTestEventsResultsPage`, would it be possible for this to indicate that it's for a specific facility like `getFacilityTestEventsResults`? Also think the purpose of the functions would be easier to parse at a glance. Totally fine if this needs to wait until cleanup.